benchmarks/test_haloprofile.py

```diff
@@ -14,11 +14,12 @@
     w0=-1,
     wa=0,
     m_nu=0,
-    m_nu_type='normal',
+    m_nu_type="normal",
     Neff=3.046,
     Omega_k=0,
-    transfer_function='eisenstein_hu',
-    mass_function='shethtormen')
+    transfer_function="eisenstein_hu",
+    mass_function="shethtormen",
+)
```

Comment:
If these closing parentheses at the end were at the end of the last argument, the code would still abide by the `flake8` rules. Closing brackets in Python don't need to be C-like. That's just a suggestion. :)